ts/features/wallet/common/BasePaymentMethodScreen.tsx

```diff
@@ -1,136 +1,146 @@
 import * as pot from "@pagopa/ts-commons/lib/pot";
-import { Button } from "native-base";
+import { useNavigation } from "@react-navigation/native";
 import * as React from "react";
-import { View, StyleSheet } from "react-native";
-import { connect } from "react-redux";
-import { Dispatch } from "redux";
+import { GestureResponderEvent, StyleSheet, View } from "react-native";
+import { ScrollView } from "react-native-gesture-handler";
+import LoadingSpinnerOverlay from "../../../components/LoadingSpinnerOverlay";
+import { useIOToast } from "../../../components/Toast";
 import { VSpacer } from "../../../components/core/spacer/Spacer";
-import { Label } from "../../../components/core/typography/Label";
 import { IOColors } from "../../../components/core/variables/IOColors";
-import { IOStyles } from "../../../components/core/variables/IOStyles";
-import LoadingSpinnerOverlay from "../../../components/LoadingSpinnerOverlay";
-import DarkLayout from "../../../components/screens/DarkLayout";
+import {
+  IOStyles,
+  IOVisualCostants
+} from "../../../components/core/variables/IOStyles";
+import BaseScreenComponent from "../../../components/screens/BaseScreenComponent";
+import FocusAwareStatusBar from "../../../components/ui/FocusAwareStatusBar";
+import ListItemAction from "../../../components/ui/ListItemAction";
 import I18n from "../../../i18n";
-import { navigateBack } from "../../../store/actions/navigation";
 import { deleteWalletRequest } from "../../../store/actions/wallet/wallets";
-import { GlobalState } from "../../../store/reducers/types";
+import { useIODispatch, useIOSelector } from "../../../store/hooks";
 import { getWalletsById } from "../../../store/reducers/wallet/wallets";
 import { PaymentMethod } from "../../../types/pagopa";
 import { emptyContextualHelp } from "../../../utils/emptyContextualHelp";
-import { showToast } from "../../../utils/showToast";
 import { useRemovePaymentMethodBottomSheet } from "../component/RemovePaymentMethod";
 
-type OwnProps = {
+type Props = {
   paymentMethod: PaymentMethod;
   card: React.ReactNode;
   content: React.ReactNode;
 };
 
-type Props = ReturnType<typeof mapDispatchToProps> &
-  ReturnType<typeof mapStateToProps> &
-  OwnProps;
-
 const styles = StyleSheet.create({
   cardContainer: {
-    height: 235,
+    height: 207,
     width: "100%",
+    paddingHorizontal: IOVisualCostants.appMarginDefault,
     position: "absolute",
-    top: 16,
-    zIndex: 7,
-    elevation: 7,
+    bottom: -50,
     alignItems: "center"
   },
-  headerSpacer: {
-    height: 172
-  },
-  cancelButton: {
-    borderColor: IOColors.red,
-    width: "100%"
+  blueHeader: {
+    marginBottom: IOVisualCostants.appMarginDefault,
+    marginTop: -300,
+    height: 470,
```

Comment:
Too much spacing on top. Maybe a little less?

```suggestion
    height: 460,
```

Reply:
@mastro993 @forrest57  Ideally, these values should be derived from other values (Percentage of `deviceHeight`? Percentage of card height? Etc...) because they seem too fragile for other devices that are not our simulator. If you have any doubts about the design logic, ask our designer colleagues for clarification.